org.garudalinux.firedragon.yml

```diff
@@ -0,0 +1,173 @@
+app-id: org.garudalinux.firedragon
+name: FireDragon
+runtime: org.freedesktop.Platform
+runtime-version: 23.08
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: 23.08
+    add-ld-path: .
+command: firedragon
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.firedragon
+  - --filesystem=home/.local/share/applications:create
+  - --filesystem=home/.local/share/icons:create
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --filesystem=xdg-run/speech-dispatcher:ro
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=org.mpris.MediaPlayer2.firefox.*
+  - --own-name=org.mozilla.firedragon.*
```

Comment:
you should own this dbus name since you are not mozilla.

Reply:
FireDragon is a fork of Floorp which in itself is a fork of Firefox. Firefox uses the `org.mozilla.` prefix with the configured app name supplied during compilation. We just kept the same behavior as Floorp. https://github.com/flathub/one.ablaze.floorp/blob/7bf909d6777b933be2d69fe848bcbd9b3613ed34/one.ablaze.floorp.yml#L41

Reply:
Yea forks usually don't change the bus name prefix, the last component gets changed based on the remoting name.